lib/cuckoo/common/objects.py

```diff
@@ -361,6 +361,8 @@ def get_type(self):
         """
         if self.file_type:
             return self.file_type
+        else:
+            self.file_type = self.get_content_type()
```

Comment:
technically, it's not necessary to put this in an 'else' block.

Reply:
i have really no idea why Kevin changed that so i will leave this PR to be handle by him